packages/@angular/cli/tasks/init.ts

```diff
@@ -72,15 +72,14 @@ export default Task.extend({
     });
 
     const cwd = this.project.root;
-    const schematicName = CliConfig.fromGlobal().get('defaults.schematics.newApp');
```

Comment:
No need to retrieve config value from multiple places. Now it's always responsibility of the caller.